ExternalProjectHelpers.cmake

```diff
@@ -87,6 +98,12 @@ endif()
 set(source-cache "${CMAKE_BINARY_DIR}/sourcecache" CACHE FILEPATH "Directory for cached source downloads")
 file(MAKE_DIRECTORY ${source-cache})
 
+set(build-cache "" CACHE FILEPATH "Directory for cached builds (to avoid rebuilding already built dependencies)")
+set(BIOFORMATS_EP_BUILD_CACHE "${build-cache}")
+
+set(python-cache "" CACHE FILEPATH "Directory for cached python builds (to avoid rebuilding already built build dependencies)")
```

Comment:
No, it can be removed.  Can it wait for a future PR for that though?

`python-cache` will likely need renaming anyway--the distinction is really between runtime dependencies and build-time dependencies where `build-cache` is the former and `python-cache` is tha latter; once we have non-python build-time dependencies it will need adjusting.